src/ucs/config/parser.c

```diff
@@ -1956,5 +1954,5 @@ UCS_STATIC_CLEANUP {
         ucs_free((void*)key);
         ucs_free(value);
     })
-    kh_destroy_inplace(ucs_config_map, &ucs_config_file_vars);
+        kh_destroy_inplace(ucs_config_map, &ucs_config_file_vars);
```

Comment:
why reverted?
should make right indent even if clang-format can't recognize it..

Reply:
By mistake. Reverted back.